_data/changelogs/component-header.yml

```diff
@@ -2,6 +2,17 @@ title: Header
 type: component
 changelogURL:
 items:
+  - date: 2024-10-04
+    summary: Removed the CSS order property from mobile header navigation.
+    summaryAdditional: |
+      Now, the visual order of the component matches the tab order.
+      This is marked as a breaking change, but you only need to change your project code if you want to keep the search bar at the top of the menu.
```

Comment:
@thisisdano Can you confirm that this summary works for you? [Link to related discussion](https://github.com/uswds/uswds-site/pull/2798#discussion_r1786416549)